woof-code/rootfs-petbuilds/fixmenusd/fixmenusd.c

```diff
@@ -9,6 +9,10 @@
 #include <signal.h>
 #include <sys/stat.h>
 #include <fcntl.h>
+#include <regex.h>
+#include <stdio.h>
+
+#define SPOTLIST "^(firefox|firefox-[a-z]+|google-chrome-[a-z]+|chromium|vivaldi-[a-z]+|brave-browser|transmission-[a-z]+|seamonkey|sylpheed|claws-mail|thunderbird|vlc|steam)$"
```

Comment:
Tested Chrome (the official .deb, manually extracted to /), Vivaldi (the official .deb, manually extracted to /), Chromium (apt), Thunderbird (apt) and VLC (apt).

I also tested installation of Chrome and Vivaldi using petget, by clicking the .deb file. PPM alters the .desktop file and the application is still successfully configured to run as spot, as expected.